src/content/protocols/aave.md

```diff
@@ -0,0 +1,916 @@
+---
+protocol: "Aave v3"
+website: "https://aave.com"
+x: "https://x.com/aave"
+github: ["https://github.com/aave-dao/aave-v3-origin"]
+defillama_slug: ["aave-v3"]
+chain: "Ethereum"
+stage: 0
+risks: ["L", "H", "H", "H", "L"]
+reasons: []
+author: ["sagaciousyves"]
+submission_date: "1970-01-01"
+publish_date: "1970-01-01"
+update_date: "1970-01-01"
+---
+
+# Summary
+
+Aave v3 is a lending protocol that allows users to lend and borrow different ERC20 assets. Users are able to create positions that consist of debt in different loan assets which is secured by different collateral assets. The lending market allows anyone to liquidate insolvent positions, based on an external price feed and specific collateral factors representing an asset's specific risk profile. Furthermore, instead of borrowing supplied assets, Aave V3 also issues its own stablecoin, `GHO`. Users can borrow and lend `GHO` like any other asset in the system.
+
+The Aave DAO is Aave's onchain governance system, allowing `AAVE`, `stkAAVE` and `aAAVE` holders to govern over various aspects ranging across treasury management, risk management and strategic initiatives.
+
+Different Aave protocol _Instances_ exist and are managed by the Aave DAO. These instances focus on specific use cases and chains. This review covers the Ethereum Mainnet Core instance.
+
+# Risks
+
+## Chain
+
+The report is concerned with the Aave V3 _Core_ instance deployed on Ethereum mainnet. Ethereum mainnet achieves a _Low_ centralization risk score.
+
+> Chain score: Low
+
+## Upgradeability
+
+a
+The Aave v3 protocol can be organized in a number of logical modules or subsystems: _Core Lend & Borrow_, _Reserve Parameters_, _Treasury, Aave Ecosystem Reserves & Rewards_, _GHO Stablecoin_ and _Aave Governance_. Each of these subsystems exposes various degrees of control as explained in more detail below. Overall, these control vectors can result in the loss of user funds, loss of unclaimed yield or otherwise materially affect the expected performance of the protocol.
```

Comment:
lets use "modules" instead of "subsystems" throughout the review as its shorter

Reply:
at the end of the paragraph write: "materially affect the expected performance of the protocol and thus achieve a High centralization risk score."